CodeConverter/CSharp/CommonConversions.cs

```diff
@@ -403,7 +446,8 @@ private static bool ContextHasIdenticalDefaults(TokenContext context, TokenConte
             var remainingModifiers = modifiers.ToList();
             if (declaredAccessibility != Accessibility.NotApplicable) {
                 remainingModifiers = remainingModifiers.Where(m => !m.IsVbVisibility(false, false)).ToList();
-                foreach (var visibilitySyntaxKind in CsSyntaxAccessibilityKindForContext(declaredAccessibility, context)) {
+                foreach (var visibilitySyntaxKind in
+                    CsSyntaxAccessibilityKindForContext(declaredAccessibility, context)) {
```

Comment:
I'd be happy with a PR that wraps everything to 200 characters. 120 character wrapping randomly mixed with code changes is hard to review even with your pointers in the description

Reply:
Could you undo these wrapping changes so I can review the PR properly please? (At least ones that you can easily revert where no functionality changed)

Reply:
Yes I will undo them tomorrow. Btw is the standard in this repo wrap at 200? I could edit the .editorconfig then.

Reply:
There isn't currently a standard in the repo, but yes let's set that as the standard - feel free to set it to that in the .editorconfig.
Hopefully that leaves auto-wrapping as a rare last resort and most of the times humans will format things in the way they feel is most readable.

Reply:
i reverted the formatting changes and set the wrap limit at 200 chars.